src/NUnitFramework/framework/Constraints/FloatingPointNumerics.cs

```diff
@@ -45,9 +44,8 @@ namespace NUnit.Framework.Constraints
     ///     as low as 0.0000001 for small numbers or as high as 10.0 for large numbers.
     ///   </para>
     /// </remarks>
-    public class FloatingPointNumerics
+    internal static class FloatingPointNumerics
```

Comment:
Due to the inaccessible constructor, this class was already effectively `static` from the outside.

Reply:
It looks like lots of this class isn't used by the framework. I'm not sure on it's origins - are the public methods designed to be used by end users, or has internal usage of it just lapsed? 

The former would be a reason not to make it internal, of course. `static` seems fine to me.